<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsWindow</class>
 <widget class="QMainWindow" name="SettingsWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>380</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background: #0F0F12;
border: 0;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">background: #1A1A1F;</string>
   </property>
   <widget class="QFrame" name="titleFrame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>500</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>500</width>
      <height>30</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>30</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background: #1A1A1F;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Shadow::Raised</enum>
    </property>
    <widget class="QLabel" name="windowTitleLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>200</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #E5E7EB;</string>
     </property>
     <property name="text">
      <string>Settings</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="closeButton">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>0</y>
       <width>30</width>
       <height>30</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Font Awesome 7 Free</family>
       <pointsize>12</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	color: #E5E7EB;
	border: 0;
}

QPushButton:hover {
	background: #EF4444;
}</string>
     </property>
     <property name="text">
      <string>ÔÄç</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="mainFrame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>500</width>
      <height>350</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: #0F0F12;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Shadow::Raised</enum>
    </property>
    <widget class="QLabel" name="pathLabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>460</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #E5E7EB;</string>
     </property>
     <property name="text">
      <string>Default Download Path:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="pathEdit">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>380</width>
       <height>35</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit {
        background: #1A1A1F;
        color: #E5E7EB;
        border: 1px solid #27272F;
        border-radius: 5px;
        padding: 0 5px;
      }
      QLineEdit:focus {
        border: 1px solid #FF4D6D;
      }</string>
     </property>
     <property name="placeholderText">
      <string>Select folder...</string>
     </property>
    </widget>
    <widget class="QPushButton" name="browseButton">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>50</y>
       <width>70</width>
       <height>35</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
        background: #27272F;
        color: #E5E7EB;
        border-radius: 5px;
      }
      QPushButton:hover {
        background: #3A3A45;
      }</string>
     </property>
     <property name="text">
      <string>Browse</string>
     </property>
    </widget>
    
    <widget class="QLabel" name="updateLabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>100</y>
       <width>460</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #E5E7EB;</string>
     </property>
     <property name="text">
      <string>Updater:</string>
     </property>
    </widget>

    <widget class="QPushButton" name="updateButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>130</y>
       <width>460</width>
       <height>35</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
        background: #27272F;
        color: #E5E7EB;
        border-radius: 5px;
      }
      QPushButton:hover {
        background: #3A3A45;
      }</string>
     </property>
     <property name="text">
      <string>Update yt-dlp</string>
     </property>
    </widget>

    <widget class="QTextEdit" name="updateLog">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>180</y>
       <width>460</width>
       <height>100</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Consolas</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QTextEdit {
        background: #1A1A1F;
        color: #A0A0A0;
        border: 1px solid #27272F;
        border-radius: 5px;
      }</string>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="placeholderText">
      <string>Update logs will appear here...</string>
     </property>
    </widget>

    <widget class="QPushButton" name="saveButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>295</y>
       <width>460</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
        background: #FF4D6D;
        color: #1A1A1F;
        border-radius: 5px;
      }
      QPushButton:hover {
        background: #FF1F4B;
      }</string>
     </property>
     <property name="text">
      <string>Save Settings</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
